src/Microsoft.Azure.SignalR.Management/WebsocketsHubLifetimeManager.cs

```diff
@@ -30,7 +32,8 @@ public Task UserAddToGroupAsync(string userId, string groupName, CancellationTok
             }
 
             var message = new UserJoinGroupMessage(userId, groupName).WithTracingId();
-            return ServiceConnectionContainer.WriteAsync(message);
+            Log.StartToAddUserToGroup(Logger, message);
+            return WriteAsync(message);
```

Comment:
`return ServiceConnectionContainer.WriteAsync(message);` -> `return WriteAsync(message);`
is it expected?

Reply:
Yes, `WriteAsync(message)` calls `ServiceConnectionContainer.WriteAsync(message);` inside, just add some logs
@vwxyzh